lizzy_client/cli.py

```diff
@@ -186,20 +215,20 @@ def create(definition: str, image_version: str, keep_stacks: Optional[int],
         with Action('Requesting traffic change..'):
             try:
                 lizzy.traffic(stack_id, traffic)
-            except requests.ConnectionError as e:
+            except requests.ConnectionError as e:  # pragma: no cover
```

Comment:
Maybe we could create a section into `.setup.cfg` file `[coverage:report]` with `exclude_lines` directive to avoid all those comments? http://coverage.readthedocs.io/en/coverage-4.0.3/excluding.html#advanced-exclusion